src/app/showcase/components/message-popup/showcase-message-popup.component.ts

```diff
@@ -34,4 +35,15 @@ export class ShowcaseMessagePopupComponent {
 				console.log('Observable returned to showcase', v);
 			});
 	}
+
+	public showCustomized() {
```

Comment:
There's no need to mark this as public, it's set by default

Reply:
Ok, thanks for the advice